globals.go

```diff
@@ -390,6 +395,8 @@ var (
 		HostStarting,
 		HostProvisioning,
 		HostProvisionFailed,
+		HostStopping,
+		HostStopped,
```

Comment:
Just verifying, have you looked at where UpHostStatus and DownHostStatus are called to ensure that behavior will be still be as expected when HostStopping/HostStopped are included?

Reply:
Looked at each status group and UpHostStatus and ActiveStatus were the only ones where including HostStopping/HostStopped made sense. UpHostStatus is necessary for some FindHostsById functions, and ActiveStatus is necessary for some stats that count hosts by distro.